src/libslic3r/PrintConfig.cpp

```diff
@@ -189,6 +189,14 @@ static t_config_enum_values s_keys_map_WallSequence {
 };
 CONFIG_OPTION_ENUM_DEFINE_STATIC_MAPS(WallSequence)
 
+//Orca
+static t_config_enum_values s_keys_map_WallDirection{
+    { "default", int(WallDirection::Default) },
```

Comment:
I suggest we use 'auto' for clearer clarification.
What do you think?

Reply:
Good idea. Done.